.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Build with HEMTT
         uses: arma-actions/hemtt@master
         with:
-          command: 'build --release'
+          command: build --release --ci
```

Comment:
Was it not detecting CI support

Reply:
Seems it wasn't. It started doing the checks and prebuild after adding this.